revenuecatui/src/iosMain/kotlin/com/revenuecat/purchases/kmp/ui/revenuecatui/UIKitPaywall.kt

```diff
@@ -103,3 +106,28 @@ private fun UIView.getIntrinsicContentSizeOfFirstSubView(): Int? =
  * "reserve" a spot in the Compose slot table.
  */
 private class ViewControllerWrapper(var value: RCPaywallViewController?)
+
+/**
+ * Uses NonCooperative UIKitInteropInteractionMode if available, for snappy scrolling.
+ */
+@OptIn(ExperimentalComposeUiApi::class)
+private fun nonCooperativeUiKitInteropPropertiesNonExperimental(): UIKitInteropProperties =
+    try {
+        // This UIKitInteropProperties constructor and the UIKitInteropInteractionMode type are
+        // experimental at the time of writing. We try to use them, but this could fail if we're
+        // linked against a newer version of CMP that changed this API. If so, we fall back to the
```

Comment:
I think you meant `KMP`?
```suggestion
        // linked against a newer version of KMP that changed this API. If so, we fall back to the
```

Reply:
Good eye, but I actually mean CMP, for Compose Multiplatform 😄 That's the library that this API is coming from.

Reply:
Yes indeed, better safe than sorry! Thanks for paying attention 😄